dev/devicelab/lib/framework/framework.dart

```diff
@@ -170,8 +171,12 @@ class _TaskRunner {
       print(stackTrace);
       return TaskResult.failure('Task timed out after $taskTimeout');
     } finally {
-      await checkForRebootRequired();
-      await forceQuitRunningProcesses();
+      if (runProcessCleanup) {
```

Comment:
I had to squash the ~19 commits to make the rebase a bit more manageable.

This if statement was the only thing added, so tests wouldn't timeout building the flutter tool.